src/NuGet.Core/NuGet.Packaging/Signing/Timestamp/Rfc3161TimestampProvider.cs

```diff
@@ -196,7 +214,14 @@ private static void ValidateTimestampResponse(byte[] nonce, byte[] messageHash,
             {
                 throw new TimestampException(NuGetLogCode.NU3019, Strings.SignError_TimestampIntegrityCheckFailed);
             }
+        }
 
+        /// <summary>
+        /// Returns the FriendlyName of an Oid. If FriendlyName is null, then the Oid string is returned.
+        /// </summary>
+        private static string GetNameOrOidString(Oid oid)
+        {
+            return oid.FriendlyName?.ToUpper() ?? oid.Value;
```

Comment:
Why force uppercase?

Reply:
To keep it uniform with out algorithm names, which are in all upper case.